core/ledger/util/couchdb/couchdb_test.go

```diff
@@ -74,9 +75,10 @@ func TestMain(m *testing.M) {
 
 func testMain(m *testing.M) int {
 	// Switch to CouchDB
-	address, cleanup := couchdbtest.CouchDBSetup("", "")
-	testAddress = address
-	defer cleanup()
+	if testAddress == "" {
+		testAddress, cleanupCouchDB = couchdbtest.CouchDBSetup((nil))
```

Comment:
I thought that you were planning to move couchdb setup outside testmain... like other packages